src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageComparison.java

```diff
@@ -120,4 +123,13 @@ public boolean areImagesTheSame(BufferedImage image1, BufferedImage image2) {
     }
     return true;
   }
+
+  private boolean colorsAreSimilar(Color c1, Color c2){
+    double distance = Math.pow(c1.getRed() - c2.getRed(), 2) + Math.pow(c1.getGreen() - c2.getGreen(), 2) + Math.pow(c1.getBlue() - c2.getBlue(), 2);
+    if (Math.sqrt(distance) < ACCEPTABLE_COLOR_DISTANCE){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1126)